mne/decoding/tests/test_time_gen.py

```diff
@@ -424,4 +424,23 @@ def test_decoding_time():
     tg = TimeDecoding(clf=clf, cv=cv)
     tg.fit(epochs, y=y)
 
+
+def test_string_scorer():
+    """Test whether passing string as scorer works as intended"""
+    epochs = make_epochs()
```

Comment:
These epochs are a bit long for this purpose. Crop them to get 2 time samples max.